README.md

```diff
@@ -1,295 +1,314 @@
+It provides a number of convenience features for developers, that make it simple to do the following essential tasks:
+* Creating signed security requests for API initialization, and
+* Interacting with the Data API.
 
-You can build and install the SDK directly from a Git clone, with
+For example, the SDK helps with creating a signed request for Learnosity:
 
-    git clone https://github.com/Learnosity/learnosity-sdk-ruby/
-    cd learnosity-sdk-ruby/
-    bundle install
-    rake build
-    gem install --user-install pkg/learnosity-sdk-0.2.1.gem
+![image-signed-request-creation.png](docs/images/image-signed-request-creation.png)
 
-If `bundle` is missing, you can install it with
+Once the SDK has created the signed request for you, your app sends that on to an API in the Learnosity cloud, which then retrieves the assessment you are asking for, as seen in the diagram below:
 
-    gem install --user-install bundler
+![image-assessment-retrieval.png](docs/images/image-assessment-retrieval.png)
 
-## Usage
+This scenario is what you can see running in the Quick start guide example ([see below](#quick-start-guide)).
 
-Some usage examples can be found in the `examples/` subdirectory.
+There's more features, besides. See the detailed list of SDK features on the [reference page](REFERENCE.md).
 
-### Init
+[(Back to top)](#table-of-contents)
 
-This class generates and signs init options for all supported APIs. Its
-constructor takes four mandatory arguments, and one optional argument.
+## Requirements
+1. Runtime libraries for Ruby installed. ([instructions](https://www.ruby.org/downloads/))
 
-- `service`: the name of the API to sign initialisation options for,
-- `security_packet`: a hash with at least the `consumer_key`, optionally the
-  `domain` and `timestamp`; `user_id` is also mandatory for Questions API,
-- `consumer_secret`
-- `request`: the request you want to get a signature for
-- `action` [Data API only]: the action of your request, either `get` or `post`
+2. The [Gem]() package manager installed. You use Gem to access the Learnosity Ruby SDK on [RubyGems](https://pypi.org/).
 
-The `Init#generate` method can then be used to generate the options. By default,
-it will generate a JSON string. It however takes one parameter, `encode` which,
-if false, will simply return a native Ruby Hash with the signed options.
+Not using Ruby? See the [SDKs for other languages](https://help.learnosity.com/hc/en-us/sections/360000194318-Server-side-development-SDKs).
 
-Given your consumer key and secret, and the request you want to run against,
-say, the `items` API, you just need to instantiate the
-`Learnosity::Sdk::Request::Init`, and call its `generate` method.
+### Supported Ruby Versions
+The following Ruby versions are tested and supported:
 
-```ruby
-require "learnosity/sdk/request/init"
+* Ruby 3.0.2 ??? (are we testing for this? What can we recommend here?)
+* Ruby 2.7.4 ??? (are we testing for this? What can we recommend here?)
```

Comment:
Let's only say that we support upstream-supported version. That would be “normal maintenance” and “security maintenance” from this page https://www.ruby-lang.org/en/downloads/branches/

Reply:
I said this: 
The Ruby SDK supports the “normal maintenance” and “security maintenance” versions listed on the [Ruby home page](https://www.ruby-lang.org/en/downloads/branches/). Please contact our support team if you are having trouble with a specific version. 
In this commit: https://github.com/Learnosity/learnosity-sdk-ruby/commit/95a7bf77437675aa6cd2188b72169bfad08f0b74